tools/pipelines/templates/build-npm-package.yml

```diff
@@ -407,6 +421,33 @@ extends:
                         inputs:
                           sourceFolder: '${{ parameters.buildDirectory }}/nyc/report'
                           targetFolder: $(Build.ArtifactStagingDirectory)/codeCoverageAnalysis
+                      - task: Bash@3
+                        displayName: Report Code Coverage Comparison
+                        condition: and(succeededOrFailed(), eq('${{ parameters.reportCodeCoverageComparison }}', true), eq(variables['ReportDirExists'], 'true'), eq(variables['Build.Reason'], 'PullRequest'), eq(variables['System.PullRequest.TargetBranch'], 'main'))
```

Comment:
Are the extra conditions still needed with the reportCodeCoverageComparison parameter?

Reply:
`eq(variables['Build.Reason'], 'PullRequest')`

that one seems to be duplicated.

`eq(variables['System.PullRequest.TargetBranch'], 'main')`

Could that be integrated into the parameter instead? In build-client.yml?

Reply:
Yeah, the first one is duplicated. Removed that.

Reply:
Added this in reportCodeCoverageComparison itself.